app/src/androidTest/java/com/sdp13epfl2021/projmag/ProjectCreationActivityTest.kt

```diff
@@ -1,91 +1,152 @@
 package com.sdp13epfl2021.projmag
 
 import android.content.Context
+import android.content.Intent
 import android.net.Uri
 import android.widget.Button
 import android.widget.MediaController
 import android.widget.VideoView
+import androidx.test.core.app.ApplicationProvider
 import androidx.test.espresso.Espresso.onView
-import androidx.test.espresso.action.ViewActions
-import androidx.test.espresso.action.ViewActions.click
-import androidx.test.espresso.action.ViewActions.replaceText
+import androidx.test.espresso.action.ViewActions.*
 import androidx.test.espresso.matcher.ViewMatchers.withId
 import androidx.test.ext.junit.rules.ActivityScenarioRule
-import androidx.test.platform.app.InstrumentationRegistry
-import org.junit.Before
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.google.firebase.auth.FirebaseAuth
+import com.google.firebase.auth.FirebaseUser
```

Comment:
Two imports were missing.
```suggestion
import com.google.firebase.auth.FirebaseUser
import com.sdp13epfl2021.projmag.activities.FormHelper
import com.sdp13epfl2021.projmag.activities.ProjectCreationActivity
```